source/Flotsam.cpp

```diff
@@ -118,6 +118,14 @@ void Flotsam::Move(vector<Visual> &visuals)
 
 
 
+// Pull this flotsam toward a tractor beam.
+void Flotsam::Tractor(Point pull)
+{
+	velocity = pull;
```

Comment:
Stream idea: heavier flotsams are harder to pull.
```suggestion
	double mass = count * (outfit ? outfit->Mass() : 1.);
	velocity = pull / mass;
```